pyproject.toml

```diff
@@ -17,6 +17,7 @@ readme = "README.md"
 python = ">=3.9"
 typer = ">=0.3.2"
 click = ">=8.0.3"
+numpy = ">=1.17.3"
```

Comment:
It might be best to lock numpy to 1.17.3 in our lock file like [this](https://github.com/gymrek-lab/TRTools/pull/223/files#diff-3161f66a61b9ed68303607f1dbdce1c7bdcc69c42f8ea1c2051a07a579ebcf5dR174)

Reply:
thanks! I followed the instructions and updated to numpy 1.23.0 since this earlier version of numpy was giving me an error with poetry